docs/docs/cmd/planner/task/task-add.md

```diff
@@ -76,3 +85,15 @@ Adds a Microsoft Planner task with the name _My Planner Task_ for plan with the
 ```sh
 m365 planner task add --title "My Planner Task" --planId "8QZEH7b3wkSbGQobscsM5gADCBa" --bucketId "IK8tuFTwQEa5vTonM7ZMRZgAKdna" --assignedToUserNames "Allan.Carroll@contoso.com,Ida.Stevens@contoso.com" --dueDateTime "2021-12-16"
 ```
+
+Adds a Microsoft Planner task with the name _My Planner Task_ for plan with the ID _8QZEH7b3wkbGQobscsM5gADCBa_ and for the bucket with the ID _IK8tuFTwQEa5vTonM7ZMRZgAKdna_. The new task will be assigned to the users _Allan.Carroll@contoso.com_ and _Ida.Stevens@contoso.com_ who will appear first with the asssignee priority _ !!_ 
+
+```sh
+m365 planner task add --title "My Planner Task" --planId "8QZEH7b3wkSbGQobscsM5gADCBa" --bucketId "IK8tuFTwQEa5vTonM7ZMRZgAKdna" --assignedToUserNames "Allan.Carroll@contoso.com,Ida.Stevens@contoso.com" --asssigneePriority " !!"
+```
```

Comment:
I also have an error surrounding the assigneePriority option here. I cannot execute this sample (even with single quotes):

```sh
Error: The request is invalid:
Specified order hint is in an invalid format: The input ( !!) contains an invalid exclamation point at index 2.
```

Reply:
That's weird, I could update the docs to use only a single exclamation ` !`.

Reply:
Is the explanation correct then? And dont you have the same issue?

Reply:
Single one seems to be working.

![afbeelding](https://user-images.githubusercontent.com/38426621/174497977-564f10e9-001f-446a-8c78-1ec44f1ff42a.png)

Definitely not an expert in order hints, so I can't explain why the double one doesn't work. 😅